admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UserDetailsServiceImpl.java

```diff
@@ -306,6 +306,7 @@ public UserDetailsCenterMapping createUser(UserDetailsDto userDetailsDto) {
 			}
 
 			validateMappingData(userDetailsDto.getId(), regCenters.get(0).getZoneCode(), userDetailsDto.getLangCode());
+			userDetailsDto.setId(userDetailsDto.getId().toLowerCase());

```

Comment:
When lowercase conversion is taken care of in the queries. why are we converting to lowercase here?

Reply:
this change was done such that all the entries will be in lowercase only and It'll inturn help in minimizing the SQL query time complexity.